Content.Server/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -18,26 +18,27 @@ namespace Content.Server.GameObjects
     /// A component that handles receiving damage and healing,
     /// as well as informing other components of it.
     /// </summary>
-    public class DamageableComponent : Component, IDamageableComponent
+    public class DamageableComponent : SharedDamageableComponent, IDamageableComponent
     {
         /// <inheritdoc />
         public override string Name => "Damageable";
 
-        /// <inheritdoc />
-        public override uint? NetID => ContentNetIDs.DAMAGEABLE;
-
         /// <summary>
         /// The resistance set of this object.
         /// Affects receiving damage of various types.
         /// </summary>
         [ViewVariables]
         public ResistanceSet Resistances { get; private set; }
 
-        Dictionary<DamageType, int> CurrentDamage = new Dictionary<DamageType, int>();
-        Dictionary<DamageType, List<int>> Thresholds = new Dictionary<DamageType, List<int>>();
+        public Dictionary<DamageType, int> CurrentDamage = new Dictionary<DamageType, int>();
```

Comment:
Not sure that it's a good idea to make this field public. Wouldn't some proper encapsulation be better?